algorithms/set/find_keyboard_row.py

```diff
@@ -0,0 +1,26 @@
+"""
+Given a List of words, return the words that can be typed using letters of
+alphabet on only one row's of American keyboard.
+
+For example:
+Input: ["Hello", "Alaska", "Dad", "Peace"]
+Output: ["Alaska", "Dad"]
+
+Reference: https://leetcode.com/problems/keyboard-row/description/
+"""
+def find_keyboard_row(words):
+    """
+    :type words: List[str]
+    :rtype: List[str]
+    """
+    keyboard = [
+        set(list('qwertyuiop')),
```

Comment:
I think `set('qwertyuiop')` will be fine. No need for `list`.

Reply:
```
>>> set('acvvvv')
set(['a', 'c', 'v'])
```
yep

Reply:
Okay, I will fix now